portfolio/src/main/webapp/mapScript.js

```diff
@@ -28,7 +28,7 @@ function showMarkers() {
   });
 }
 
-function initMap(markerFlag = true) {
+function initMap(showMarkersFlag = true) {
```

Comment:
optional: I think `showMarkers` is sufficient as a name. You don't usually need to specify `Flag` or other type indicators in variable names.